src/node_http_parser.cc

```diff
@@ -205,10 +380,23 @@ class Parser : public AsyncWrap, public StreamListener {
   SET_SELF_SIZE(Parser)
 
   int on_message_begin() {
+    /*
```

Comment:
Could we plz use C++ style comments everywhere inside this PR? Refs: https://github.com/nodejs/node/blob/master/doc/contributing/cpp-style-guide.md#c-style-comments.

Reply:
👍